features/payment/managing_payments/browsing_payments.feature

```diff
@@ -17,3 +17,10 @@ Feature: Browsing payments
         When I browse payments
         Then I should see a single payment in the list
         And the payments of the "#00000001" order should be "New" for "amba@fatima.org"
+
+    @ui
+    Scenario: Payments are sorted by newest as default
+        Given there is an "#00000002" order with "Apple" product
+        When I browse payments
+        Then I should see payment for the "#00000001" order as 1st in the list
```

Comment:
```suggestion
        Then I should see payment for the "#00000002" order as 1st in the list
```

As we should show `newest` as the first one. Same below

Reply:
This could be spotted on screens as well, that oldest are displayed as the first onest.